code/src/java/pcgen/rules/context/AbstractReferenceContext.java

```diff
@@ -147,8 +148,12 @@ public void initialize()
 
 	public boolean validate(UnconstructedValidator validator)
 	{
-		return getAllManufacturers().stream()
-                            .allMatch(ref -> ref.validate(validator));
```

Comment:
This is NOT logically identical because the stream does NOT guarantee it evaluates all elements.  We MUST do that to avoid crashes in the core